### PR TITLE
Add assembly_line.yml as an alias for al_package.yml

### LINKED ISSUE #38: Discussion: Revisit how we broke up different YAML files when we have had some time to get used to them
Maybe in April?

### PR DESCRIPTION
**Pull request by nonprofittechy**
Fix #38 

Don't think there are any other changes to the file layout that we have come up with in the last 12 months. This is an easy one that does help make the API for using AssemblyLine in your interview a little bit easier to follow.

### CODE REVIEW

File: docassemble/AssemblyLine/data/questions/al_package_unstyled.yml [hunk 1]
```diff
@@ -1,20 +1,4 @@
-comment: |
-  The purpose of this YAML file is to allow you to make use of all of the Assembly
-  Line standard questions, etc. with one include, but eliminate all code that 
-  changes the visual appearance of your interview.
+# DEPRECATED - use assembly_line_unstyled instead
```
Comment (BryceStevenWilley) [inline docassemble/AssemblyLine/data/questions/al_package_unstyled.yml:1]: Don't think you included this file in the branch yet

Comment (BryceStevenWilley) [reply docassemble/AssemblyLine/data/questions/al_package_unstyled.yml:1]: Everytime, you fix it the moment I comment. :laughing:

Comment (nonprofittechy) [reply docassemble/AssemblyLine/data/questions/al_package_unstyled.yml:1]: Caught it at the same moment again :)

Comment (nonprofittechy) [reply docassemble/AssemblyLine/data/questions/al_package_unstyled.yml:1]: I think I have a bad habit of opening the PR to double-check what the branch contains. I should just use compare without clicking the "open PR" button.

File: docassemble/AssemblyLine/data/questions/al_package.yml [hunk 2]
```diff
@@ -1,3 +1,7 @@
+################
+# Deprecated as of 1/11/2022
+# When enough packages point to assembly_line.yml, this file should turn into an alias
+# that just includes assembly_line.yml
```
Comment (BryceStevenWilley) [inline docassemble/AssemblyLine/data/questions/al_package.yml:4]: Any reason that you made the includes chain different between AL and AL unstyled?

* `assembly_line.yml` -> `al_package.yml` -> everything
vs
* `al_package_unstyled.yml` -> `assemble_line_unstyled.yml` -> everything

The latter seems easier, and isolates the deprecated `al_package.yml` a bit more. I don't think it breaks anything either.

Comment (nonprofittechy) [reply docassemble/AssemblyLine/data/questions/al_package.yml:4]: My reasoning was that al_package_unstyled.yml is only referenced in a very small number of interviews, while al_package is referenced in many files. Adding the extra include adds a ms or two of load time to the interview, so I was trying to reduce impact on interviews "in the wild". But a few ms isn't a big deal. I can be persuaded to handle both the same way.

Comment (BryceStevenWilley) [reply docassemble/AssemblyLine/data/questions/al_package.yml:4]: That makes sense. You could say that the extra ms or so is encouragement to switch to the faster loading include :wink:

Comment (nonprofittechy) [reply docassemble/AssemblyLine/data/questions/al_package.yml:4]: Fair enough! Maybe I'll switch it to the more "correct" way now instead of waiting.

**Review by BryceStevenWilley — APPROVED** [review]

**Pull request merged**